exercises/E-conditionals/exercise.js

```diff
@@ -8,6 +8,11 @@
 
 var name = "Daniel";
 var danielsRole = "mentor";
+if ((danielsRole = "mentor")) {
```

Comment:
Hey Rafael :wave: 

Try changing the variable `danielsRole` to `"student"` and see what the console prints.
Can we figure out why and how to fix it?